modules/wri_program/config/install/field.field.node.program_center.field_show_more_link.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 2d1fbdeb-fbfc-4fbf-9e95-c3d696bd7668
```

Comment:
These shouldn't have uuids! Adding uuids to install configs can mess up site installs, so no bueno! Let's take them all out.

Reply:
Grrr, thought I'd pulled all of these! Did a search through custom and removed all of the 'line 1' uuids.